Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/AddMockIdentitiesRepository.cs

```diff
@@ -11,6 +11,6 @@ public Task Add(Identity identity, CancellationToken cancellationToken)
     {
         WasCalled = true;
         WasCalledWith = identity;
-        return Task.FromResult(WasCalledWith);
+        return Task.FromResult(identity);
```

Comment:
This makes as much sense as the version you had before...
If the method defines a `Task` as a return value, why would you return a `Task<Identity>`?!?!